netgrasp/netgrasp.py

```diff
@@ -3,16 +3,18 @@
 from utils import email
 from utils import simple_timer
 from utils import pretty
-from config import config
+from utils import mac as libmac
+from config import config as libconfig
```

Comment:
if there's a naming conflict here, let's just rename the files rather than importing as